kerchunk/grib2.py

```diff
@@ -582,3 +584,68 @@ def correct_hrrr_subhf_step(group: Dict) -> Dict:
     group["refs"]["step/0"] = enocded_val
 
     return group
+
+
+def parse_grib_idx(
+    fs: fsspec.AbstractFileSystem,
+    *,
```

Comment:
Do you know what `*` does here? Is it reasonable?

Reply:
It's for the keyword based parameters. I assumed to pass the arguments after `fs` as keyword parameters.

Reply:
but why? The filename is *required*, and now all calling code needs to spell out `basename=` on every call.

> Is it reasonable?

Reply:
Ok, I'll remove it.